docs/site/migration/boot-scripts.md

```diff
@@ -6,8 +6,279 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-boot-scripts.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3957](https://github.com/strongloop/loopback-next/issues/3957)
-" %}
+In LoopBack 3, predefined boot scripts are organized in the `/server/boot`
+directory, and are executed right before the server starts to perform some
+custom application initialization. The same functionality can also be achieved
+in LoopBack 4 application by adding observers.
+
+The LoopBack 3 boot script is mapped to LoopBack 4 observer's `start` function.
+And moreover, besides participating in an application's start phase, an observer
+also allows you to perform logic in the stop phase.
+
+## LoopBack 3 Boot Script
+
+LoopBack 3 supports synchronous and asynchronous boot scripts. And you can
+choose either callback or promise based implementation for asynchronous scripts.
+They all have the first parameter as `server` (some people are used to name it
+as `app`). The callback based asynchronous script has a 2nd parameter `done` as
+the callback function.
+
+A typical synchronous boot script that prints out a datasource's name before
```

Comment:
Maybe say `LoopBack 3 supports three styles of boot scripts` and make each of the examples below a bullet item.